interactive_engine/common/src/main/java/com/alibaba/maxgraph/common/util/IrSchemaParser.java

```diff
@@ -65,7 +65,9 @@ public String parse(GraphSchema graphSchema) {
                         true,
                         "is_column_id",
                         true);
-        return JSON.toJson(schemaMap);
+        String json = JSON.toJson(schemaMap);
+        // logger.info("graph schema is {}, ir schema is {}", graphSchema.formatJson(), json);
+        return json;
```

Comment:
Delete the debugging logs if you won't need it anymore.

If you still need it in some cases, you could use `logger.debug`.